Presentation.Web/Tests/PageObjects/it-system/Usage/Tabs/ItSystemUsageGDPR.po.ts

```diff
@@ -28,6 +28,18 @@ class ItSystemUsageGDPR {
         return element(this.cssHelper.byDataElementType(this.consts.dataLevelTypeLegalCheckbox));
     }
 
+    static getPersonalCprCheckBox() {
+        return element(this.cssHelper.byDataElementType(this.consts.personalCprCheckbox));
+    }
+
+    static getPersonalOtherPrivateCheckBox() {
+        return element(this.cssHelper.byDataElementType(this.consts.personalOtherPrivateCheckbox));
```

Comment:
why data element type... use Id in stead. data-element-type only makes sense if the specific entries are unknown such as a column in a table set. In this case all elements are known, so data-element-type is just noise